api/users/views.py

```diff
@@ -288,25 +287,23 @@ class UserNodes(JSONAPIBaseView, generics.ListAPIView, UserMixin, ODMFilterMixin
     view_category = 'users'
     view_name = 'user-nodes'
 
+    ordering = ('-date_modified')
+
     # overrides ODMFilterMixin
     def get_default_odm_query(self):
         user = self.get_user()
         return (
-            Q('contributors', 'eq', user) &
+            Q('contributors', 'contains', user._id) &
+            Q('is_folder', 'ne', True) &
             Q('is_collection', 'ne', True) &
-            Q('is_deleted', 'ne', True)
+            Q('is_deleted', 'ne', True) &
+            Q('is_registration', 'ne', True)
```

Comment:
>  Is this a query that can be overriden by clients using the querystring? Or is this a baseline query that clients may only *further* constrain?

Looks like it's a [baseline query](https://github.com/CenterForOpenScience/osf.io/blob/35596266100a1c7848a936d59296bde2fb27b5a1/api/base/filters.py#L290).

> I ask because we advertise in the docstring that `registration` is filterable, and if the latter then this would seem to be a breaking change.

What's our policy on backwards compatibility in the API? Do we care that we're removing functionality from our documented API within a major version number (v2)?